docs/userguides/projects.md

```diff
@@ -64,7 +64,7 @@ project.MyContract.deploy(sender=account)
 
 ### Dependencies
 
-To set up dependencies in your ``ape-config.yaml`` file, follow [this guide](https://docs.apeworx.io/ape/stable/userguides/config.html#dependencies).
+To set up dependencies in your `ape-config.yaml` file, follow [this guide](https://docs.apeworx.io/ape/stable/userguides/config.html#dependencies).
```

Comment:
Will these work propertly with the documentation generator?

Reply:
The double tick marks are only needed in the doc strings. They were probably included here in the guides because of habit formed while working on the doc strings.

Reply:
Nice! So the formatter found these issues?

Reply:
Yes, it did!